TravelBug/src/Components/VideoCard/VideoCard.css

```diff
@@ -21,3 +21,7 @@
 .image {
   border-radius: 1rem;
 }
+
+.video-card:hover {
```

Comment:
The pseudo class is not necessary here. .video-card {cursor: pointer} works just fine.

Reply:
Oh simple thing but I did'nt know this . Thanks